src/strpak.cpp

```diff
@@ -406,15 +407,19 @@ char* FC strpathparts( 	// Build string from parts of a path name (for default f
 
 // Returns assembled string, equested parts which are not present in *path are omitted from result.
 {
-	char pbuf[_MAX_PATH*4];
+	char pbuf[ FILENAME_MAX*4];
 	static char wantmask[4] = { STRPPDRIVE, STRPPDIR, STRPPFNAME, STRPPEXT };
 
-#define part(p) (pbuf+((p)*_MAX_PATH))
+#define part(p) (pbuf+((p)*FILENAME_MAX))
```

Comment:
I guess all these FILENAME_MAX will have to become CSE_MAX_PATH when [292](https://github.com/cse-sim/cse/pull/292#discussion_r1010859745) is merged - maybe we can finish up this PR as a dependency of 292.

Reply:
Yes all of these FILENAME_MAX should go away once #292 is merged.